03-Custom-Actions-In-Python/03-Custom-Action-Features.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>Do we really need to demonstrate all possible combinations here? In a tutorial format it is often best to show the most common cases and leave the user to read the reference documentation to discover the other permutations. Keep in mind that users <em>will</em> copy and paste this code and expect it to just work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>